demisto_sdk/commands/format/update_playbook.py

```diff
@@ -173,6 +173,12 @@ def remove_copy_and_dev_suffixes_from_subplaybook(self):
                 task['task']['name'] = task['task'].get('name').replace('_dev', ''). \
                     replace('_copy', '')
 
+    def remove_copy_and_dev_suffixes_from_subscripts(self):
+        for task_id, task in self.data.get('tasks', {}).items():
+            if task['task'].get('scriptName'):
+                task['task']['scriptName'] = task['task'].get('scriptName').replace('_dev', ''). \
+                    replace('_copy', '')
```

Comment:
Somewhat duplicate to `self.remove_copy_and_dev_suffixes_from_str`, please consider defining it under `tools` and using it both here and in `update_layout`